src/System.Reflection.Emit.ILGeneration/tests/CustomAttributeBuilder/CustomAttributeBuilderCtor1.cs

```diff
@@ -0,0 +1,147 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+
+using System.Reflection.Emit;
+using System.Linq;
+using Xunit;
+
+namespace System.Reflection.Emit.ILGeneration.Tests
+{
+    public class CustomAttributeBuilderCtor1
+    {
+        [Fact]
+        public void PosTest1()
+        {
+            string str = "PosTest1";
+            Type[] ctorParams = new Type[] { typeof(string) };
+            object[] paramValues = new object[] { str };
+            CustomAttributeBuilder cab = new CustomAttributeBuilder(
+                typeof(ObsoleteAttribute).GetConstructor(ctorParams), paramValues);
+
+            Assert.NotNull(cab);
```

Comment:
Yeah, it can't.  We should actually be asserting that the CAB is configured correctly from its ctor, right?